tests/stats_extlog.rs

```diff
@@ -66,16 +75,42 @@ struct ThirdExternalLog {
 #[derive(ExtLoggable, Clone, Serialize)]
 #[LogDetails(Id = "4", Text = "Some more irrelevant text", Level = "Info")]
 #[StatTrigger(StatName = "test_double_grouped", Action = "Incr", Value = "1")]
-#[StatTrigger(StatName = "test_latest_foo_error_count", Condition = "self.error != 0",
-              Action = "SetVal", ValueFrom = "self.foo_count")]
+#[StatTrigger(
+    StatName = "test_latest_foo_error_count",
+    Condition = "self.error != 0",
+    Action = "SetVal",
+    ValueFrom = "self.foo_count"
+)]
```

Comment:
Did rustfmt make this change?  Feels weird to split a `#[]` atrribute over multiple lines, but I guess it's OK.

Reply:
Yes, this is apparently how rustfmt likes it.